moped-editor/src/queries/subprojects.js

```diff
@@ -2,14 +2,19 @@ import { gql } from "@apollo/client";
 
 export const SUBPROJECT_QUERY = gql`
   query SubprojectSummary($projectId: Int) {
-    subprojects: moped_project(where: { project_id: { _eq: $projectId } }) {
-      moped_projects(where: { is_deleted: { _eq: false } }) {
-        project_name
-        project_id
-        status_id
-        current_phase
+    subprojects: moped_project(
+      where: { parent_project_id: { _eq: $projectId }, is_deleted: { _eq: false } }
+    ) {
+      project_name
+      project_id
+      moped_proj_phases(where: { is_current_phase: { _eq: true } }) {
+        moped_phase {
+          phase_name
+          phase_key
+        }
       }
     }
+    # todo: we might shouldnt be fetching all projects every time this tab loads...
```

Comment:
@johnclary make a tech debt issue for this

Reply:
in working on this PR, part of me wondered if we should use a global context for lookup values. Statuses, team members, entities....

Reply:
ah, good call about fetch policy. i'll write up an issue!